rskj-core/src/test/java/co/rsk/peg/bitcoin/PocSighashTest.java

```diff
@@ -473,6 +473,73 @@ void test_each_input_sighash_is_unique_for_a_signed_erp_tx_testnet() {
         }
     }
 
+    @Test
+    void test_redeemScript_can_be_obtained_from_input() {
+        // Arrange
+        NetworkParameters networkParameters = BridgeTestNetConstants.getInstance().getBtcParams();
+        int erpFedActivationDelay = 720;
```

Comment:
```suggestion
        int erpFedActivationDelay = bridgeTestnetConstants.getErpFedActivationDelay();
```

Reply:
720 is the correct value, BrigTestNetContstants has a different value(52_560). Remember here I'm using a real testnet tx. The federation spending funds was created with that erpFedActivationDelay.